Alexa.NET/Request/Scope.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Newtonsoft.Json;
+
+namespace Alexa.NET.Request
+{
+    public class Scope
+    {
+        [JsonProperty("status")]
+        public string status { get; set; }
```

Comment:
Just to be consistent with other classes can we make the public property 'Status' (capital) (keeping the JsonProperty as-is.

Reply:
Yes ! My bad sorry.